src/doc/trpl/error-handling.md

```diff
@@ -181,6 +181,8 @@ match version {
 This function makes use of an enum, `ParseError`, to enumerate the various
 errors that can occur.
 
+The [`Debug`](../core/fmt/trait.Debug.html) trait is what let's us print the enum value using the `{:?}` format operation.
```

Comment:
ah yes, and it probably should be to std, not core.

@swaroopch , up for a second PR after this one lands? heh. It's already being tested in the rollup, or i'd say fix it first